weldx/schemas/weldx.bam.de/weldx/core/transformations/coordinate_system_hierarchy-1.0.0.yaml

```diff
@@ -13,59 +13,59 @@ examples:
   -
     - An abstract example of a coordinate system hierarchy with nested subsystems and without data
     - |
-      !weldx!core/transformations/coordinate_system_hierarchy-1.0.0
+      !<asdf://weldx.bam.de/weldx/tags/core/transformations/coordinate_system_hierarchy-1.0.0>
         name: CSM
```

Comment:
This is somewhat strange. The removed tags were stored this way in the file generated with the script I posted earlier. Had to replace them for the asdf tests to work. However, the normal reading of a file is not affected.

Reply:
This is because the weldx extension sets the custom YAML tag handle in the asdf file header when creating files, see this PR: https://github.com/asdf-format/asdf/pull/963:
```yaml
#ASDF 1.0.0
#ASDF_STANDARD 1.5.0
%YAML 1.1
%TAG ! tag:stsci.edu:asdf/
%TAG !weldx! asdf://weldx.bam.de/weldx/tags/
--- !core/asdf-1.1.0

```
Our handle is defined here https://github.com/BAMWelDX/weldx/blob/b15007fef82d7b750d8ecb14e61b7579bddb4e9b/weldx/asdf/extension.py#L58

However, the YAML tag handle context is not present when `sphinx-asdf` evaluates the schema examples and cannot be set either (I am not sure how the asdf default `!` handle gets 'injected' but we would probably have to make a change to `sphinx-asdf`)

I have listed a regex to replace the `!weldx!` tags here: https://github.com/BAMWelDX/weldx/discussions/479

Reply:
good to know.